apps/opik-frontend/src/components/shared/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -28,6 +28,10 @@ const TooltipWrapper: React.FunctionComponent<TooltipWrapperProps> = ({
   defaultOpen,
   stopClickPropagation,
 }) => {
+  if (!content) {
+    return <>{children}</>;
+  }
```

Comment:
A bit scary move, probably makes sense: I'm a bit afraid it's going to shift the markup